### DISCUSSION TITLE
S-132 Keys

### DISCUSSION THREAD

**Discussion by jdourish** [thehighnibble/thehighnibble.github.io#176] [S-132 add-on for IMSAI & Cromemco replicas]
I have just finished building my S-132 kit and have fitted it to my IMSAI 8080 and am glad to report it worked first time. There does, however, seem to be a lack of documentation on using the S-132 board.
By looking thought this forum and watching videos I have picked up the following:
Ctrl-Alt+1		VT100 terminal?
Ctrl-Alt+5		Dazzle output 
Ctrl-Alt+6		System Menu
In one video he mentions access to the Setup screen by pressing the “Function” key but does not say what this is. I found you can access the Setup A screen by using the:
Alt+PRTSC
I still cannot find how to access the other setup screen (B,C) or edit them.
Is there anywhere you can get a full list of commands as am sure there are a lot more?

**Comment by nbreeden2** [comment]
I don't have my system at hand, but I believe pressing "F1" after pressing "Alt+PRTDCR" to enter the setup menu adds help information to the setup screens. (Maybe Alt-F1)?

Overall info can be found at:
https://thehighnibble.com/s132/#overview

**Reply by jdourish** [reply]
Thanks, how did I miss the functions list, RTFM.
